tests/component/system/test_device_properties.py

```diff
@@ -112,3 +112,70 @@ def test___module___get_chassis___shared_interpreter(device: Device):
     chassis = device.compact_daq_chassis_device
 
     assert chassis._interpreter is device._interpreter
+
+
+def test___ext_cal_last_date_and_time___no_errors(any_x_series_device: Device) -> None:
+    last_date_and_time = any_x_series_device.ext_cal_last_date_and_time
+
+    assert last_date_and_time is not None
+
+
+def test___self_cal_last_date_and_time___no_errors(any_x_series_device: Device) -> None:
+    last_date_and_time = any_x_series_device.self_cal_last_date_and_time
+
+    assert last_date_and_time is not None
+
+
+def test___device_supports_cal___no_errors(any_x_series_device: Device) -> None:
+    is_cal_supported = any_x_series_device.device_supports_cal
+
+    assert is_cal_supported is True
+
+
+def test___cal_acc_connection_count___no_errors(any_x_series_device: Device) -> None:
+    test_value = 99
+
+    try:
+        current_acc_connection_count = any_x_series_device.cal_acc_connection_count
+
+        any_x_series_device.cal_acc_connection_count = test_value
+        assert test_value == any_x_series_device.cal_acc_connection_count
+
+        any_x_series_device.cal_acc_connection_count = current_acc_connection_count
+
+    except DaqError as e:
+        if e.error_code != DAQmxErrors.ATTR_NOT_SUPPORTED:
+            raise
+
+
+def test___cal_dev_temp___no_errors(any_x_series_device: Device) -> None:
+    temperature = any_x_series_device.cal_dev_temp
+
+    assert 0.0 != temperature
```

Comment:
I think simulated returns 0.0 by looking at the code... make this a real_x_series maybe? Or maybe I'm just looking in the wrong place?

Reply:
wait no, last temp is 0. I think this might be something else.

Reply:
I did a quick test, simulated device will return 0; whereas real device will return its current temperature reading. This may vary on different devices and different environment temperature. I'll use `sim_x_series_device` for this test case then.